app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,32 @@
+class PostsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @post = Post.new
+    @posts = Post.all.order('created_at DESC')
+  end
+
+  def create
+    @user = current_user
```

Comment:
to keep the controller skinny, you could maybe explore passing the user_id into the post_params method as a hidden form element?

Reply:
nice suggestion, thanks